org.lflang/src/org/lflang/validation/LFValidator.xtend

```diff
@@ -83,7 +83,7 @@ import org.lflang.lf.WidthSpec
 
 import static extension org.lflang.ASTUtils.*
 import static extension org.lflang.JavaAstUtils.*
-import org.lflang.federated.SupportedSerializers
+import org.lflang.federated.serialization.SupportedSerializers
```

Comment:
Should serialization be part of the `federated` package? As you mentioned this morning, we might want to employ serialization to enable communication between reactors of different targets without relying to inter-process communication.

Reply:
That's an excellent point.

I think for the time-being, the serialization package is mainly focused on federated execution. Once we are able to generalize its functionality, we should definitely consider moving it out of the federated package.